### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by d-hayashi**
## What?
This patch includes the following updates:
- Bump eslint-plugin-compat from 3.9.0 to 3.11.1 #104
- Bump eslint-config-standard from 16.0.2 to 16.0.3 #103
- Bump eslint-plugin-react from 7.23.2 to 7.24.0 #100
- Bump react-dom from 17.0.1 to 17.0.2 #98
- Bump @testing-library/user-event from 13.1.5 to 13.2.1 #97

## Why?
To update dependencies

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/105#" title="Author self-approved">d-hayashi</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by hdl-service** [comment]
[Keywords](https://help.github.com/articles/closing-issues-using-keywords) which can automatically close issues and at(@) or hashtag(#) mentions are not allowed in commit messages.

**The list of commits with invalid commit messages**:

- [a44b5b7](https://github.com/dataware-tools/app-data-browser-next/commits/a44b5b78f08c6cd1fb8f93469a338db3d4703cc8) Bump @testing-library/user-event from 13.1.5 to 13.2.1

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by d-hayashi** [comment]
@WatanabeToshimitsu 
こちらで動作確認した限りでは問題なさそうだったので、ざっと見て問題無さそうであればマージお願いします！

**Review by WatanabeToshimitsu — APPROVED** [review]
うぉお....
これはめっちゃありがたいです.....

そろそろUIのビジュアルリグレッションテスト仕込んで，こういうのを自動で出来るようにしたいですね

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/105#" title="Author self-approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/105#pullrequestreview-715715695" title="Approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [WatanabeToshimitsu,d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
